app/components/forms/wizard/basic-details-step.js

```diff
@@ -91,6 +91,121 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return validationRules;
   }),
 
+  ticketsValidationRules() {
```

Comment:
For what good is frontend validation when the server is still accepting such data. 
There should be server checks for such empty fields, Also I am not sure that we need to roll back to native jquery validation. 
I will wait for other mentors to review on this one.
cc: @iamareebjamal @uds5501 @mrsaicharan1